client/src/modules/cohort/model/actions.js

```diff
@@ -48,6 +54,74 @@ const updateCohortFailure = err => ({
   payload: err.message
 });
 
+const archiveCohortSuccess = data => ({
+  type: CohortActionTypes.ARCHIVE_SUCCESS,
+  payload: data
+});
+
+const archiveCohortFailure = errMessage => ({
+  type: CohortActionTypes.ARCHIVE_FAILURE,
+  payload: errMessage
+});
+
+const archiveCohortRequest = () => ({
+  type: CohortActionTypes.ARCHIVE_REQUEST
+});
+
+const restoreCohortSuccess = (data, cohortId) => ({
+  type: CohortActionTypes.RESTORE_SUCCESS,
+  payload: { data, cohortId }
+});
+
+const restoreCohortFailure = errMessage => ({
+  type: CohortActionTypes.RESTORE_FAILURE,
+  payload: errMessage
+});
+
+const restoreCohortRequest = () => ({
+  type: CohortActionTypes.RESTORE_REQUEST
+});
+
+const deleteCohortSuccess = cohortId => ({
+  type: CohortActionTypes.DELETE_SUCCESS,
+  payload: cohortId
+});
+
+const deleteCohortFailure = errMessage => ({
+  type: CohortActionTypes.DELETE_FAILURE,
+  payload: errMessage
+});
+
+const deleteCohortRequest = () => ({
+  type: CohortActionTypes.DELETE_REQUEST
+});
+
+const fetchCohortDataFailure = errMessage => ({
+  type: CohortActionTypes.FETCH_DATA_FAILURE,
```

Comment:
the previous renaming proposal gonna be related with renaming this across many files :D I'm sorry  😃 
But It's only my proposal - you can disagree 👎